bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -111,7 +111,7 @@ <h4 class="mzp-c-wordmark mzp-t-wordmark-sm {% if mozilla_monitor %}t-product-mo
           </h4>
         </a>
         <ul class="mzp-u-list-styled">
-          <li>{{ ftl('firefox-accounts-get-email-alerts') }}</li>
+          <li>{% if LANG.startswith('en-') %}Get automated removal of your exposed info from sites that are selling it{% else %}{{ ftl('firefox-accounts-get-email-alerts') }}{% endif %}</li>
```

Comment:
Shouldn't this be `en-US`, since automatic data removal is not available in Canada/UK?

Reply:
Ah seemed like I missed this. I can get this patched up 👍🏼